2021-08-26 06:59:03,853 INFO    MainThread:20289 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 06:59:03,854 INFO    MainThread:20289 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 06:59:03,854 INFO    MainThread:20289 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_065903-1ablfu0r/logs/debug.log
2021-08-26 06:59:03,854 INFO    MainThread:20289 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_065903-1ablfu0r/logs/debug-internal.log
2021-08-26 06:59:03,854 INFO    MainThread:20289 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9389d84280>
2021-08-26 06:59:03,855 INFO    MainThread:20289 [wandb_init.py:init():375] calling init triggers
2021-08-26 06:59:03,855 INFO    MainThread:20289 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'epochs': 3, 'batch_size': 32, 'learning_rate': 0.001}
2021-08-26 06:59:03,855 INFO    MainThread:20289 [wandb_init.py:init():395] re-initializing run, found existing run on stack: vtudew78
2021-08-26 06:59:03,857 INFO    MainThread:20289 [wandb_run.py:finish():1254] finishing run presto105/lv1_p/vtudew78
2021-08-26 06:59:03,857 INFO    MainThread:20289 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 0
2021-08-26 06:59:03,857 INFO    MainThread:20289 [wandb_run.py:_restore():1526] restore
2021-08-26 06:59:04,686 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 1081
}

2021-08-26 06:59:04,923 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 1081
}

2021-08-26 06:59:05,660 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 10284
}

2021-08-26 06:59:05,763 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 10284
}

2021-08-26 06:59:05,867 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 10284
}

2021-08-26 06:59:05,969 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 10284
}

2021-08-26 06:59:06,071 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 06:59:06,173 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 06:59:06,275 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 06:59:06,377 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 06:59:06,479 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 06:59:06,580 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 06:59:06,774 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 06:59:06,876 INFO    MainThread:20289 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10284
  total_bytes: 10284
}

2021-08-26 06:59:08,012 INFO    MainThread:20289 [wandb_run.py:_show_files():1898] logging synced files
2021-08-26 06:59:08,017 INFO    MainThread:20289 [wandb_init.py:init():424] starting backend
2021-08-26 06:59:08,017 INFO    MainThread:20289 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-26 06:59:08,020 INFO    MainThread:20289 [backend.py:ensure_launched():135] starting backend process...
2021-08-26 06:59:08,052 INFO    MainThread:20289 [backend.py:ensure_launched():139] started backend process with pid: 21654
2021-08-26 06:59:08,055 INFO    MainThread:20289 [wandb_init.py:init():429] backend started and connected
2021-08-26 06:59:08,073 INFO    MainThread:20289 [wandb_run.py:_label_probe_notebook():798] probe notebook
2021-08-26 06:59:08,073 INFO    MainThread:20289 [wandb_run.py:_label_probe_notebook():805] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-26 06:59:08,073 INFO    MainThread:20289 [wandb_init.py:init():477] updated telemetry
2021-08-26 06:59:08,074 INFO    MainThread:20289 [wandb_init.py:init():500] communicating current version
2021-08-26 06:59:09,300 INFO    MainThread:20289 [wandb_init.py:init():505] got version response 
2021-08-26 06:59:09,300 INFO    MainThread:20289 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-26 06:59:09,520 INFO    MainThread:20289 [wandb_init.py:init():540] starting run threads in backend
2021-08-26 06:59:14,090 INFO    MainThread:20289 [wandb_run.py:_console_start():1584] atexit reg
2021-08-26 06:59:14,090 INFO    MainThread:20289 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-08-26 06:59:14,090 INFO    MainThread:20289 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-08-26 06:59:14,091 INFO    MainThread:20289 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-26 06:59:14,091 INFO    MainThread:20289 [wandb_init.py:init():565] run started, returning control to user process
